src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -1174,6 +1253,35 @@ LRESULT CALLBACK WindowImplWin32::globalOnEvent(HWND handle, UINT message, WPARA
     return DefWindowProcW(handle, message, wParam, lParam);
 }
 
+////////////////////////////////////////////////////////////
+void WindowImplWin32::prepareTouch()
+{
+    static bool prepared = false;
+    if (!prepared)
+    {
+        prepared = true;
+
+        // Reset touch IDs
+        for (int i = 0; i < 10; ++i)
+            touchIDs[i] = static_cast<DWORD>(-1);
```

Comment:
Maybe `std::fill()`?

What is the `10` magic number?

Reply:
Leftover from before, but yeah, should be a constant. It's the maximum number of simultaneous touch inputs Windows supports, which I think is still 10.